libs/ballot-interpreter-vx/src/interpreter/index.ts

```diff
@@ -699,6 +752,16 @@ export class Interpreter {
       }
     }
 
+    // if we couldn't find a close enough alignment, the skew is too much and we bail
+    if (
+      !bestMatchNewTemplatePixels ||
```

Comment:
Are meaning to throw an error if `bestMatchNewTemplatePixels` is `0` or `undefined` or both? Just making sure because it can be dangerous to use `!` on a number

Reply:
great catch! only if undefined, I will correct. It's unlikely to ever be zero, but... yeah